src/lipalightninglib.udl

```diff
@@ -8,6 +8,10 @@ enum Network {
 dictionary Config {
     Network network;
     sequence<u8> seed;
+    string esplora_api_host;
+    u16? esplora_api_port;
+    string? esplora_api_path;
+    boolean esplora_over_tls;
```

Comment:
Agreed. The `Url`-option is probably not really feasible, because `Config` is an interface that is being exposed through UniFFI, which can only expose primitives AFAIK.

So I'll go ahead and fit it all into a simple String.

Reply:
It's feasible. We can have the `Config` custom struct hold another custom struct like e.g. the `Url`. But the String is simpler and is fine by me too.

Reply:
Ah, yes. But we either can expose the public methods in UniFFI or create our own Url struct and have complete freedom.

Reply:
Yeah we can @andrei-21. But feels like a bloat. Let's KISS